lib/server/monalisa-synchronization/MonAlisaSynchronizer.js

```diff
@@ -71,13 +71,22 @@ class MonAlisaSynchronizer {
                 }
             };
 
+            const errors = {};
             for (const monalisaDataPass of dataPasses) {
                 await dataSource.transaction(() => transactionablePipeline(monalisaDataPass))
-                    .catch((error) => this.logger.errorMessage(error.message, {}));
+                    .catch((error) => {
```

Comment:
You are mixing await with `.catch`, it's very strange. Please use only await.

Reply:
applied